app/Console/Commands/MaintenanceModeQueueCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Jobs\BringSiteBackUp;
+use Illuminate\Console\Command;
+
+class MaintenanceModeQueueCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'maintenance:queue';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Put the site into maintenance mode and bring it back up after 5 minutes';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        // Put the site into maintenance mode
+        $this->call('down');
+
+        // Schedule bringing the site back up after 5 minutes
+        BringSiteBackUp::dispatch()->delay(now()->addMinutes(5));
```

Comment:
I know you mentioned in your [comment on the ticket](https://github.com/accessibility-exchange/platform/issues/1867#issuecomment-1814977282) that lifecycle hooks aren't possible at the moment in your infrastructure. I'm just wondering if there really isn't a way to know when the old pod is removed and to trigger the `artisan up` call then. Unfortunately adding a delay doesn't adequately address the issue.

Reply:
We weren't able to come up with an alternative.